packages/react/src/components/PageHeader/PageHeader.stories.js

```diff
@@ -91,11 +94,164 @@ export default {
+          venenatis turpis. Porttitor urna mollis semper commodo! Vehicula
+          ridiculus himenaeos justo ut in platea cras facilisis potenti nostra
+          facilisis? Orci habitant, ac himenaeos volutpat. Montes nullam fusce
+          habitasse semper primis et! Iaculis praesent sit platea nam tortor?
+          Penatibus quam tristique tempus vulputate facilisis dapibus dapibus
+          ornare accumsan, urna dictumst. Posuere nascetur feugiat velit felis
+          auctor aenean sociosqu bibendum? Turpis diam enim bibendum curae;
+          viverra a orci dignissim montes ac.
+        </p>
+        <p>
+          Ornare ac ligula accumsan aptent etiam ut auctor ut commodo erat.
+          Convallis ad donec tempus montes lacus volutpat quisque congue.
+          Gravida sodales pellentesque sociis orci ut mauris semper auctor odio
+          nulla. Natoque posuere phasellus consectetur sed semper non aptent!
+          Placerat eros laoreet consectetur eros molestie eros nec nisi vivamus
+          morbi? Torquent cras ultricies ut pretium urna platea nam adipiscing
+          consequat. Iaculis egestas at malesuada id dapibus rhoncus sapien at?
+          Accumsan laoreet per primis elementum metus magnis dis tristique
+          consectetur.
+        </p>
+      </TabPanel>
+    ),
+  },
+  {
+    label: 'Tab 2',
+    panel: (
+      <TabPanel key={1}>
+        <p>Tab 2</p>
+      </TabPanel>
+    ),
+  },
+  {
+    label: 'Tab 3',
+    panel: (
+      <TabPanel key={2}>
+        <p>Tab 3</p>
+      </TabPanel>
+    ),
+    disabled: true,
+  },
+  {
+    label: 'Tab 4',
+    panel: (
+      <TabPanel key={3}>
+        <p>Tab 4</p>
+      </TabPanel>
+    ),
+  },
+  {
+    label: 'Tab 5',
+    panel: (
+      <TabPanel key={4}>
+        <p>Tab 5</p>
+      </TabPanel>
+    ),
+  },
+  {
+    label: 'Tab 6',
+    panel: (
+      <TabPanel key={5}>
+        <p>Tab 6</p>
+      </TabPanel>
+    ),
+  },
+  {
+    label: 'Tab 7',
+    panel: (
+      <TabPanel key={6}>
+        <p>Tab 7</p>
+      </TabPanel>
+    ),
+  },
+  {
+    label: 'Tab 8',
+    panel: (
+      <TabPanel key={7}>
+        <p>Tab 8</p>
+      </TabPanel>
+    ),
+  },
+  {
+    label: 'Tab 9',
+    panel: (
+      <TabPanel key={8}>
+        <p>Tab 9</p>
+      </TabPanel>
+    ),
+  },
+  {
+    label: 'Tab 10',
+    panel: (
+      <TabPanel key={9}>
+        <p>Tab 10</p>
+      </TabPanel>
+    ),
+  },
+];
+
 export const Default = (args) => (
   <PageHeader.Root {...args}>
     <PageHeader.BreadcrumbBar />
     <PageHeader.Content />
-    <PageHeader.TabBar />
+    <PageHeader.TabBar>
+      <PageHeader.Tabs onTabCloseRequest={() => {}}>
+        <TabList>
+          {tabs.map((tab, index) => (
+            <Tab key={index} disabled={tab.disabled}>
+              {tab.label}
+            </Tab>
+          ))}
+        </TabList>
+        <TabPanels>{tabs.map((tab) => tab.panel)}</TabPanels>
```

Comment:
I think the `.map` is more than we need for these stories. 


```suggestion
        <TabList>
          <Tab>Dashboard</Tab>
          <Tab>Monitoring</Tab>
          <Tab>Activity</Tab>
          <Tab>Settings</Tab>
        </TabList>
        <TabPanels>
          <TabPanel>Dashboard Tab Panel</TabPanel>
          <TabPanel>Monitoring Tab Panel</TabPanel>
          <TabPanel>Activity Tab Panel</TabPanel>
          <TabPanel>Settings Tab Panel</TabPanel>
        </TabPanels>
```